snudda/neuromodulation/neuromodulation_synapse.py

```diff
@@ -0,0 +1,374 @@
+from snudda.simulate.simulate import SnuddaSimulate
+import json
+import numpy as np
+import snudda.neuromodulation.modulation as modulation
+import snudda.neuromodulation.translator as translator
+from snudda.simulate.nrn_simulator_parallel import NrnSimulatorParallel
+from snudda.utils.snudda_path import snudda_parse_path
+import h5py
+from snudda.neurons.neuron_model_extended import NeuronModel
+from snudda.utils.load import SnuddaLoad
+from neuron import h
+
+
+class SnuddaNeuromodulationSynapse(SnuddaSimulate):
+
+    def __init__(self,
+                 network_path=None,
+                 network_file=None,
+                 input_file=None,
+                 verbose=False,
+                 log_file=None,
+                 disable_gap_junctions=True,
+                 simulation_config=None, neuromodulators=None, neuromodulator_description=None,neuromodulation_conductance=None):
+
+        self.input_data = h5py.File(snudda_parse_path(input_file), 'r')
+        self.verbose = verbose
+        self.neuromodulators = neuromodulators
+        self.neuro_desc = neuromodulator_description
+        self.neuromodulation = dict()
+        self.current_cell = None
+        self.syn_gpcrs = list()
+        self.cell_modulator = dict()
+        self.neuromodulation_conductance = neuromodulation_conductance
+
+        super(SnuddaNeuromodulationSynapse, self).__init__(network_path=network_path,
+                                                    network_file=network_file,
+                                                    input_file=input_file,
+                                                    verbose=False,
+                                                    log_file=log_file,
+                                                    disable_gap_junctions=disable_gap_junctions,
+                                                    simulation_config=simulation_config)
+
+
+    def reorder_cell_info(self, cell_modulator):
```

Comment:
Varför behövs det? Byter du plats på allt, eller kan något missas?

Reply:
rename to organise

Reply:
och mer kommentarer

Reply:
skriv att du måste anropa organise.....gpcr för att allt ska instanticeras